packages/strapi-generate-new/lib/utils/usage.js

```diff
@@ -72,7 +72,7 @@ function trackError({ scope, error }) {
         nodeVersion: process.version,
         docker: scope.docker,
         useYarn: scope.useYarn,
-        template: scope.template || '',
+        packageJsonStrapi: scope.packageJsonStrapi,
```

Comment:
Is this `packageJsonStrapi` key handled by our tracker? I thought the [tracker PR](https://github.com/strapi/tracker/pull/21/files) only added support a `starter` key

Reply:
good point and good catch! I'm going to double check with Alex but I think you are probably right the tracker PR needs to be updated.